metadata-ingestion/src/datahub/entrypoints.py

```diff
@@ -201,3 +204,6 @@ def main(**kwargs):
         if gms_config:
             logger.debug(f"GMS config {gms_config}")
         sys.exit(1)
+    finally:
+        if _logging_configured:
+            _logging_configured.__exit__(None, None, None)
```

Comment:
Does this unset `_logging_configured`? Or if not, is it ok to call `__exit__` multiple times?

Reply:
fixed